src/docs/uxd/language/DOC.md

```diff
@@ -6,4 +6,48 @@
 
 ## Date and time standards
 
-[Google doc](https://docs.google.com/document/d/1hivSfe48717VhlLRtjyqReS5twbSLuPC6MISJt3MTG4/edit?usp=sharing)
+### Displaying exact time
+
+For Cloud Services and its apps, which will not be localized, we recommend displaying exact time in UTC and displaying the date as shown below. UTC is a 24 hour clock, so takes the place of localized time display.
+**Exact date and time display-- 07 Jan 2019, 23:33 UTC**
+
+However, note that we highly recommend displaying relative time for most use cases.
+
+### Displaying relative time
+
+For the use cases we have encountered so far, we assume that it is more useful for users to be able to tell at a glance how fresh the information they are looking at is, or how recently something was added, than the exact time/date that something happened. **Using relative time ("two days ago," "5 minutes ago") allows users to quickly get a sense of how recently something happened without having to do math or reason about dates.**
+
+Referencing Stacy's [master doc](https://docs.google.com/document/d/1RA3C5Caza3boGz-uJ8b0-R7HOhNdS83SfRpcKH_5vxk/edit?usp=sharing) on time/date styles...
+
+| If it has been...    | Stacy:             | Katie's thought                     |
```

Comment:
Where it says Stacy you can change it to "PatternFly" and where it says "Katie" you can replace with "Cloud Services"